components/isceobj/StripmapProc/runDispersive.py

```diff
@@ -8,13 +8,10 @@
 from isceobj.Constants import SPEED_OF_LIGHT
 import numpy as np
 import gdal
-<<<<<<< HEAD
 from scipy.ndimage import median_filter
 from astropy.convolution import convolve
-from scipy import ndimage
 import numpy as np
```

Comment:
This import is duplicated ^

This merge conflict was probably due to https://github.com/isce-framework/isce2/pull/82 where I tried to hide the scipy/astropy imports, is it still possible to import stripmapApp.Insar without these modules? If not can the imports be hidden?